packages/astro/src/core/create-vite.ts

```diff
@@ -60,7 +61,7 @@ export async function createVite(
 		clearScreen: false, // we want to control the output, not Vite
 		logLevel: 'warn', // log warnings and errors only
 		optimizeDeps: {
-			entries: ['src/**/*'], // Try and scan a user’s project (won’t catch everything),
+			entries: ['src/pages/**/*'],
```

Comment:
just curious: why not scan everything?

Reply:
Yeah, curious about this too.

Reply:
Ah I can revert this, was just playing around with it. Could maybe help our startup time.